### ISSUE TITLE
Question: how to configure carbon

### ISSUE THREAD

**Issue by polettix** [dokku/dokku-graphite#50]
**TL;DR:** Is there any way to provide custom configuration files to Graphite/Carbon/etc.?

<hr>

I'm new to Graphite and I see that the configuration of the storage/retention part is done via Carbon, like described [here](https://graphite.readthedocs.io/en/latest/config-carbon.html).

The `graphite:info` sub-command shows a configuration directory as *Config dir*:

```
$ dokku graphite:info collector
=====> collector graphite service information
       Config dir:          /var/lib/dokku/services/grafana-graphite-statsd/collector/config
       Config options:                               
       Data dir:            /var/lib/dokku/services/grafana-graphite-statsd/collector/data
       Dsn:                 statsd://dokku-graphite-collector:8125
       Exposed ports:       8125->8130 8126->24633 80->16633 81->3218 2003->10232 
       Id:                  035c4592067a5898310fe25facaeb57b79c38341c4d3a9dac247708c8bf819fe
       Internal ip:         172.17.0.3               
       Initial network:                              
       Links:               -                        
       Post create network:                          
       Post start network:                           
       Service root:        /var/lib/dokku/services/grafana-graphite-statsd/collector
       Status:              running                  
       Version:             dokku/docker-grafana-graphite:6.4.4
```

I tried to put a modified `storage-schemas.conf` file inside the *Config dir* and then restarted the service, but it seems that the new configuration file does not end in the right place inside the container.

Is there any way to provide custom configurations for Graphite/Carbon?

In case there is and someone knows about it, I'd be happy to try and transform any quick answer into some documentation.

**Comment by nemtsov** [comment]
I believe this is a bug, the config dir is not mounted in [the functions](https://github.com/dokku/dokku-graphite/blob/master/functions#L85-L86).

PR to fix this: #80
